COPYRIGHT

```diff
@@ -0,0 +1,31 @@
+Intellectual Property Notice
```

Comment:
@white238 modified ``COPYRIGHT`` and/or ``LICENSE`` in axom to help github figure out the license.